website/src/word-panel.tsx

```diff
@@ -129,6 +126,28 @@ export const WordPanel = (p: {
   }
 }
 
+export const PartLines = (p: {
+  segments: Dailp.FormFieldsFragment["segments"]
+}) => {
+  if (p.segments) {
+    let length = p.segments.length
+    return (
+      <div>
+        {p.segments.map((part, index) => (
+          <tr>
+            <td className={css.table}>
```

Comment:
You called the style class `table` but it's for table cells. Could use a better name here.